website/docs/Queries.md

```diff
@@ -212,7 +212,9 @@ getByLabelText(
 ): ReactTestInstance;
 ```
 
-Returns a `ReactTestInstance` with matching `accessibilityLabel` prop.
+Returns a `ReactTestInstance` with matching label:
+- either by matching [`accessibilityLabel`](https://reactnative.dev/docs/accessibility#accessibilitylabel) prop
+- or by matching text content of view referenced by [`accessibilityLabelledBy`](https://reactnative.dev/docs/accessibility#accessibilitylabelledby-android) prop
```

Comment:
should we mention that this prop only works for android? it's also specified in the RN doc but I feel it's the first time an RNTL query would validate code working only on android so it might be worth pointing out

Reply:
This is a good question, whether RNTL should work like `ios`, `android`, common part of `iOS` and `android` or either `iOS` or `Android`. So far we seem to take the last approach when it comes to `isHiddenFromAccessibility`: element is hidden if any iOS or Android prop indicates that it is hidden.

Reply:
true, did not notice it back when we implemented it. For isHiddenFromAccessibility, we could add a little note in [this page](https://callstack.github.io/react-native-testing-library/docs/api#ishiddenfromaccessibility)? Or is overkill?

Reply:
It is something that would be good to analyse and discuss in the mid-future. We have various options here should figure out which one makes most sense from user POV. For now I would stick with the current approach of: "this works with either iOS or Android".